chapter3.adoc

```diff
@@ -41,6 +41,12 @@ The `relaxedpriv` field is hardwired to 0.
 
 The field `aamvirtual` in the command (at 0x17 in the Debug Module) determines whether the Access Memory command uses a physical or virtual address. When an Access Memory command is issued with `aamvirtual`=0, the hart must check whether the physical access is allowed to access memory. The hart responds with an exception to the Debug Module when M-mode is not permitted to debug, `tvm` (in mstatus) is set to 1, and `mode` (in satp) enables any kind of virtual translation. In the event of an exception, the Debug Module set `cmderr` to 3 and clear the data registers to 0.
 
+==== Quick Access 
+
+When M-mode debugging is not allowed (`mdbgen`=0) for a hart, any Quick Access operation will be discarded, causing `cmderr` being set to 6.
+
+[NOTE] Disallowing the Quick Access command for M-mode simplifies the implementation, as it avoids the need for special handling of Quick Access. Consequently, Quick Access is inhibited for debuggers that are only allowed to debug supervisor domains. 
```

Comment:
I might flesh this note out a bit more.

> [NOTE] Quick Access abstract commands effect a halt, memory or register access, and resume on the selected hart.  It is undesirable for these Quick Access halts to remain pending until debug is allowed, because the debugger blocks waiting for Quick Access completion.  Returning an error only on Quick Access commands received when debug is not allowed would require the hart to distinguish between Quick Access halt requests and other halt requests.  Because Quick Access is merely an optimized flow, not one that is required for any usage models, it was decided to avoid burdening the hart with extra hardware, and instead forbid Quick Access when `mdbgen`=0.

Reply:
Change request with extended note explains well the rationale for QA decision.